models/user.py

```diff
@@ -128,6 +129,10 @@ def find_by_role_and_name(cls, role, name):
             & (UserModel.firstName.ilike(likeName) | UserModel.lastName.ilike(likeName))
         ).all()
 
+    @classmethod
+    def find_users_without_assigned_role(cls):
+        return cls.query.filter(and_(cls.role.is_(None), cls.archived.is_(False)))
+
```

Comment:
I think this is OK, but I'd like to get this unit tested. If you'd like to do this that would be great; otherwise let's get an issue created to test all happy and sad paths. Additionally the name of the method is better, however it contradicts what it actually does. It says it will return all users with an assigned role, but it does not do that. It returns active users without an assigned role. But that doesn't really matter. The most important thing is to get some tests so that we can document what this does and build our confidence that it does what it says it does.